ui/app/models/kv/data.js

```diff
@@ -52,7 +52,12 @@ const validations = {
 @withFormFields()
 export default class KvSecretDataModel extends Model {
   @attr('string') backend; // dynamic path of secret -- set on response from value passed to queryRecord.
-  @attr('string', { label: 'Path for this secret' }) path;
+  @attr('string', {
+    label: 'Path for this secret',
+    subText:
+      'When using / in the path, the UI will interpret the words before / as the name of the directory.',
```

Comment:
This reads a little strange, and I think it might be better to just say "forward slash." also "words" isn't really accurate - since it could be a combo of numbers/letters/symbols. Brought up in slack to bring in design's input

Reply:
Amended per slack discussion.